packages/client/components/ReflectionGroup/DraggableReflectionCard.tsx

```diff
@@ -77,11 +103,15 @@ const DraggableReflectionCard = (props: Props) => {
     swipeColumn,
     dataCy
   } = props
-  const {id: meetingId, teamId, localStage} = meeting
+  const {id: meetingId, teamId, localStage, spotlightReflection} = meeting
   const {isComplete, phaseType} = localStage
   const {isDropping, isEditing} = reflection
-  const [drag] = useState(makeDragState)
+  const isSpotlightOpen = !!spotlightReflection?.id
+  const isInSpotlight = isSpotlightOpen && !openSpotlight
+  const isBehindSpotlight = isSpotlightOpen && !isInSpotlight
   const staticReflectionCount = staticReflections?.length || 0
+  const [drag] = useState(makeDragState)
+  updateSpotlightDrag(drag, isBehindSpotlight, isInSpotlight, spotlightReflection?.id)
```

Comment:
I tried passing in a few args to `makeDragState` and updating the drag object with Spotlight logic that way, but it would update too slowly and throw off the UI